server/sonar-web/src/main/js/app/components/nav/component/branch-like/CurrentBranchLike.tsx

```diff
@@ -97,7 +97,7 @@ export function CurrentBranchLike(props: CurrentBranchLikeProps) {
             data-test="branches-support-disabled"
             links={[
               {
-                href: 'https://www.sonarsource.com/plans-and-pricing/developer/',
+                href: 'https://knowledgebase.autorabit.com/codescan/docs/set-up-codescan-payment',
```

Comment:
This should redirect to https://www.codescan.io/pricing/ however it should never come up as the branches are always enabled on CS Cloud